pkg/arvo/app/metadata-store.hoon

```diff
@@ -44,11 +44,35 @@
       creator=@p
   ==
 ::
++$  metadata-1
+  $:  title=@t
+      description=@t
+      color=@ux
+      date-created=@da
+      creator=@p
+      module=term
+  ==
+::
++$  md-resource-1   [=app-name =app-path]
+::
++$  associations-1  (map [group-path md-resource-1] metadata-1)
+::
 +$  base-state-1
-  $:  associations=associations
-      group-indices=(jug group-path md-resource)
+  $:  associations=associations-1
+      group-indices=(jug group-path md-resource-1)
       app-indices=(jug app-name [group-path app-path])
-      resource-indices=(jug md-resource group-path)
+      resource-indices=(jug md-resource-1 group-path)
+  ==
+::
++$  cached-indices
+  $:  group-indices=(jug resource md-resource)
+      app-indices=(jug app-name [group=resource =resource])
+      resource-indices=(map md-resource resource)
+  ==
+::
++$  base-state-2
+  $:  =associations
+      ~
```

Comment:
Why a sig?

Reply:
idk, something makes me mildly uncomfortable about a base-state that's just an alias for a type. Happy to change if you feel strongly about this

Reply:
Nah, doesn't matter